central/auth/service/service_impl.go

```diff
@@ -61,13 +102,13 @@ func authStatusForID(id authn.Identity) (*v1.AuthStatus, error) {
 	_, notValidAfter := id.ValidityPeriod()
 	exp, err := types.TimestampProto(notValidAfter)
 	if err != nil {
-		return nil, errors.Errorf("expiration time: %s", err)
+		return nil, pkgErrors.Errorf("expiration time: %s", err)
```

Comment:
This isn't very clear. What do you want to communicate to the end user?

Reply:
I'm not willing to make this change / refactoring in this PR, as it is already quite big and has gone through enough review cycles. Additionally it should be included in the 4.3 cut-off and is currently only blocked by the failing style checks on master. 

This is an existing error and only highlighted due to changing the package name.